tests/samplers_tests/test_nsgaii.py

```diff
@@ -559,6 +577,160 @@ def _create_frozen_trial(
     return trial
 
 
+def test_child_generation_strategy_invalid_value() -> None:
+    mutation_prob_err_msg = "The value of `mutation_prob` must be in the range [0, 1]."
+    crossover_prob_err_msg = "The value of `crossover_prob` must be in the range [0, 1]."
+    swapping_prob_err_msg = "The value of `swapping_prob` must be in the range [0, 1]."
```

Comment:
I believe Optuna generally avoids testing for exact matches of detailed error messages like this. It can lead to fragile tests, so it's better to avoid them.

Reply:
I removed the check.